chainer/functions/math/trigonometric.py

```diff
@@ -100,3 +100,143 @@ def backward(self, x, gy):
 def tan(x):
     """Elementwise tan function."""
     return Tan()(x)
+
+
+class Arcsin(function.Function):
+
+    @property
+    def label(self):
+        return 'arcsin'
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        type_check.expect(in_types[0].dtype.kind == 'f')
+
+    def forward(self, x):
+        xp = cuda.get_array_module(*x)
+        return utils.force_array(xp.arcsin(x[0])),
+
+    def backward_cpu(self, x, gy):
+        gx = utils.force_array(numpy.square(x[0]))
+        numpy.negative(gx, out=gx)
+        gx += 1
+        numpy.sqrt(gx, out=gx)
+        numpy.reciprocal(gx, out=gx)
+        gx *= gy[0]
+        return gx,
+
+    def backward_gpu(self, x, gy):
+        gx = cuda.elementwise(
+            'T x, T gy', 'T gx',
+            'gx = (T)1.0 / sqrt((T)1.0 - x * x)',
+            'arcsin_bwd'
+        )(x[0], gy[0])
+        return gx,
+
+
+def arcsin(x):
+    """Elementwise arcsine function.
+
+    .. math::
+       y_i = \\arcsin x_i.
```

Comment:
I suppose the mathematical form usually do not come with parenthesis `y = arcsin x` and the current line is rendered correctly, how about it?